### PR TITLE
Fixes #16779: Update install/upgrade docs for 6.0.3

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16779

### CODE REVIEW

File: src/reference/modules/installation/pages/upgrade.adoc [hunk 1]
```diff
@@ -143,6 +132,18 @@ please first upgrade to one of the supported versions, and then upgrade to 6.0.
 
 === 5.0 servers
 
+[WARNING]
+====
+
+There was a bug in 6.0 agent before 6.0.4 preventing
+that made their inventories not recognizable from 5.0
```
Comment (fanf) [inline src/reference/modules/installation/pages/upgrade.adoc]: missing words ?

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**